app/presenters/blacklight/document_presenter.rb

```diff
@@ -93,7 +93,8 @@ def retrieve_values(field_config)
     deprecation_deprecate retrieve_values: 'Use FieldPresenter#values'
 
     def field_presenter(field_config, options = {})
-      FieldPresenter.new(view_context, document, field_config, options)
+      presenter_class = field_config.presenter || Blacklight::FieldPresenter
```

Comment:
That looks great. Can the same be done here?

Reply:
Not really. We don't have a class for just index/show fields.